Request for code Review

CornerstoneII opened the pull request:
# Request for code review.

> In this branch, we cloned an old version of the [Apple.com](https://rawcdn.githack.com/CornerstoneII/Building-with-Backgrounds-and-Gradients/0f9877fc7ef4aa2aeea2a6140a7ecc6545019669/index.html) page in line with the following requirements and features.

* Set background **image** on the **main** section page.
* Set a gradient color on the **nav** bar.
* Used **Font awesome 4 icons** and open-source images
* **image** sources from the [original](https://web.archive.org/web/20140301004610/http://www.apple.com/) page.
* **Flex** and **Floats** to achieve layouts.

CornerstoneII committed:
Test linter

CornerstoneII committed:
Set page html structure

CornerstoneII committed:
Fill in nav content

CornerstoneII committed:
Fill the text in the jumbotron section

CornerstoneII committed:
Fill in footer section

CornerstoneII committed:
Clean the css

CornerstoneII committed:
Add background-image to the main section

CornerstoneII committed:
Rename the .show-case to the aside

CornerstoneII committed:
Add rest and font style to the body

CornerstoneII committed:
Remove underline from links

CornerstoneII committed:
Create flex class and apply to nav>ul

CornerstoneII committed:
Implement the nav

CornerstoneII committed:
Add padding to nav ul

CornerstoneII committed:
Remove margin to nav and add padding main

CornerstoneII committed:
Add style to the .jumbotron section

CornerstoneII committed:
Downloaded source image

CornerstoneII committed:
Add the image sources to the html file

CornerstoneII committed:
Style the image to fit gallery

CornerstoneII committed:
Add footer style

CornerstoneII committed:
Edit padding to align BG-image with original

CornerstoneII committed:
Download icons

CornerstoneII committed:
Add apple icon and adjust nav alignment

CornerstoneII committed:
Add play icon jumbotron section

CornerstoneII committed:
Add links inside the jumbotron h1, h2, p

CornerstoneII committed:
Add the USA flag and adjust it containing element

CornerstoneII committed:
Add magnify icon to search

CornerstoneII committed:
Change the flag-icon image source

CornerstoneII committed:
Add border-right to nav li

CornerstoneII committed:
Add hover effect to the nav li

CornerstoneII committed:
Add font-weight to footer > a. Add border-right

belsman committed:
Set local linters

belsman committed:
Add padding-left to nav input

belsman committed:
Adjust Jumbotron up down padding

belsman committed:
Make nav input expandable

belsman committed:
Add black magnify on input:focus

belsman committed:
Add transition property to nav:input focus

belsman committed:
Add padding and Edit nav input:focus

belsman committed:
Add gradient color to the nav bar

belsman committed:
Add favicon

belsman committed:
Edit README.md

belsman committed:
Add App screen_shot

belsman committed:
Fix stylelint error

belsman committed:
Add live demo link

kubilaycaglayan commented on index.html:
```diff
@@ -0,0 +1,79 @@
+</head>
+
+<body>
+    <main id="hero-image">
```
Unused id attributes.

kubilaycaglayan commented on index.html:
```diff
@@ -0,0 +1,79 @@
+                <li><a href="#">iPad</a></li>
+                <li><a href="#">iTunes</a></li>
+                <li><a href="#">Support</a></li>
+                <li class="search"><input type="search" name="search" id="q"></li>
```
Unused id attributes.

kubilaycaglayan commented on index.html:
```diff
@@ -0,0 +1,79 @@
+            </h2>
+            <p class="description">
+                <a href="#" class="hover-underline">Watch the "Your verse" film and explore the stories</a>
+                <span id="play"><i class="fa fa-play-circle-o play-icon"></i></span>
```
Unused id attribute.

kubilaycaglayan reviewed (changes requested):
Hello Team :wave: 
Status: Changes required :recycle: 

Amazing project! :boom:  I loved your precision :man_health_worker:  :woman_health_worker: 

Let's cover a few details and make sure that your project is at the top mastery level! :star: 

**Pull Request**
- [x] Pull Request has a meaningful title and a brief description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
Please work on the title :pray: 

**Repository**
- [x] Files not related to the project should not be committed and added to .gitignore (e.g. `.DS_Store`) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)) 
-> package-lock.json & package.json should be inside .gitignore file.
-> And please make sure you have deleted them from your GitHub repository too.
:notebook: **Note:** In the future projects(JavaScript section) we will not put these files into the .gitignore. But for now, it seems you are only using those files for local purposes.
:notebook: **Note 2:** Adding them into .gitignore may not be enough to delete them from the GitHub repository. Please research more to achieve the goal.

**HTML & CSS**
- [x]  You have unused ids, please delete them. Details are in the comments section. (We don't need those if we are not using them in CSS, as selectors.)

This is a great project, please keep up the good work. :chart_with_upwards_trend: 
You can reach out to me via Slack. :people_holding_hands: 
With all my respect...

belsman committed:
Update .gitignore

Added the following files
  - package.json
  -package-lock.json
  -hintrc
  -stylelintrc.json

belsman committed:
Eliminate unused IDs

belsman committed:
Delete .hintrc

belsman committed:
Delete .stylelintrc.json

belsman committed:
Delete package-lock.json

belsman committed:
Delete package.json

kitpao commented on README.md:
```diff
@@ -1 +1,52 @@
+
+## Live Demo
+
+[Live Demo Link](https://rawcdn.githack.com/CornerstoneII/Building-with-Backgrounds-and-Gradients/0f9877fc7ef4aa2aeea2a6140a7ecc6545019669/index.html)
```
- [ ] **[OPTIONAL]** Your live demo link is not updated. As a suggestion, so you don't have to be manually updating this link during your code reviews, use the second link (development link) in githack. That one will automatically load your last version in the same branch, so you don't have to worry about it later :)

kitpao reviewed (approved):
# 👋 Hello @CornerstoneII and @belsman 

💯 You did an amazing job! and also implemented all the previous TSE's requirements 🌟 Congratulations 😄 

**_Note:_** You have 1 optional suggestion. Optionals are **not required** in order to approve your project but are personal suggestions to improve your coding experience. Please check below to see it 🔽 

### 👏 Your project has been approved 👏
### 🚀 You may merge to master and submit the project completition form 🚀

___
##### 👉 If you have any doubts you can contact me in slack as @kitpao

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__
